rakelib/vendor.rake

```diff
@@ -60,9 +60,12 @@ namespace "vendor" do
   end # def untar
 
   task "jruby" do |task, args|
-    name = task.name.split(":")[1]
-    info = VERSIONS[name]
+    JRUBY = "jruby"
+    JRUBY_RUNTIME = "jruby-runtime"
+
+    info = VERSIONS[JRUBY_RUNTIME] || VERSIONS[JRUBY]
```

Comment:
THis implicitly uses the "jruby-runtime" setting from the versions.yml, right? Why have two versions?

Reply:
because, as I tried to decribe in the new header comments I added for each section in the yml file, the `jruby` section is still needed for the gradle compile to fetch the required jars which are published artifacts. 

IF the `jruby-runtime` is specified, it is only use for the vendored version.